src/router-setup.js

```diff
@@ -226,6 +246,12 @@ function stopAP() {
  *                    the command.
  */
 function defineNetwork(ssid, password) {
+  DEBUG && console.log('router-setup: defineNetwork ssid:', ssid);
+
+  // TODO: eth0 is RPi specific and will be different for each potential
```

Comment:
In my opinion, the platform code should handle this itself without having to pass in the interface name. The function is called `setWanMode`, so that code should probably be able to figure out for itself what the WAN interface is.

Reply:
On the turris, there is eth0, eth1, and eth2. eth1 is the WAN. linux-openwrt.js would need to know that it's running on a turris omnia

Reply:
Maybe we put that into one of the config files and let the platform code read it.